roles/ckan/templates/kubernetes/ckan.yaml

```diff
@@ -236,9 +231,11 @@ spec:
         - name: ckan-configs
           secret:
             secretName: ckan-configs
-        - name: google-analytics-credentials
+{% if ckan_googleanalytics_enable %}
+        - name: ckan-configs
           secret:
             secretName: google-analytics-credentials
+{% endif %}
```

Comment:
I think these lines can be deleted now?  The secret is loaded as part of the ckan-configs the few lines above?

Reply:
I think they are still required because the secret is not loaded if the extension is not enabled. And of course mounting will fail is the secrets were not loaded.